pythnet/message_buffer/programs/message_buffer/src/lib.rs

```diff
@@ -18,7 +21,8 @@ pub mod message_buffer {
 
     /// Initializes the whitelist and sets it's admin to the provided pubkey
     /// Once initialized, the authority must sign all further changes to the whitelist.
-    pub fn initialize(ctx: Context<Initialize>, admin: Pubkey) -> Result<()> {
+    pub fn initialize(ctx: Context<Initialize>) -> Result<()> {
+        let admin = ctx.accounts.admin.key();
```

Comment:
I think `whitelist.admin = ctx.accounts.admin.key();` is easier to read

Reply:
done thanks